### DISCUSSION TITLE
赞！保持段落结构大赞！

### DISCUSSION THREAD

**Discussion by 6Tom** [chunibyocola/sc-translator-crx#88] [General]
又是我，这回是特意来感谢的！

刚想说，划词翻译会把换行符整丢了可咋办，4.6版就加上了，太好了，非常感谢！

我现在已经习惯原文+译文带下划线双语模式了，太赞了

**Comment by chunibyocola** [comment]
哈哈，感谢支持，很高兴能帮上忙😀
